src/osgPlugins/OpenFlight/ReaderWriterFLT.cpp

```diff
@@ -373,15 +411,18 @@ class FLTReaderWriter : public ReaderWriter
                 document.setUseTextureAlphaForTransparancyBinning(options->getOptionString().find("noTextureAlphaForTransparancyBinning")==std::string::npos);
                 OSG_DEBUG << readerMsg << "noTextureAlphaForTransparancyBinning=" << !document.getUseTextureAlphaForTransparancyBinning() << std::endl;
 
-                document.setReadObjectRecordData(options->getOptionString().find("readObjectRecordData")!=std::string::npos);
```

Comment:
I think we want this fix since it appears to be a typo fix in the 3.6 code.

Reply:
Pending.